src/sw.ts

```diff
@@ -94,7 +95,32 @@ self.addEventListener('fetch', (event) => {
     // intercept and do our own stuff...
     event.respondWith(fetchHandler({ path: url.pathname, request }))
   } else if (isSubdomainRequest(event)) {
-    event.respondWith(fetchHandler({ path: url.pathname, request }))
+    const cacheKey = event.request.url
```

Comment:
In theory we could simplify it here a bit and just use the request object rather than a custom cache key. The cache API looks at the `Vary` header when matching, but it looks like we don't set it. 

> To determine if two Requests match, the browser uses more than just the URL. Two requests are considered different if they have different query strings, Vary headers, or HTTP methods (GET, POST, PUT, etc.). https://web.dev/articles/cache-api-quick-guide#retrieving